app/app/src/main/java/org/omecproject/up4/cli/FlowsStatsCommand.java

```diff
@@ -0,0 +1,162 @@
+/*
+ SPDX-License-Identifier: Apache-2.0
+ SPDX-FileCopyrightText: 2022-present Intel Corporation
+ */
+package org.omecproject.up4.cli;
+
+import com.google.common.collect.Maps;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.omecproject.up4.impl.Up4AdminService;
+import org.onlab.packet.Ip4Address;
+import org.onosproject.cli.AbstractShellCommand;
+import org.onosproject.net.behaviour.upf.UpfCounter;
+
+import java.util.Map;
+
+/**
+ * UP4 UE flow stats.
+ */
+@Service
+@Command(scope = "up4", name = "flow-stats",
+        description = "Read all UE flows stats")
+public class FlowsStatsCommand extends AbstractShellCommand {
+
+    private static final String SEPARATOR = "-".repeat(40);
+    private static final int DEFAULT_SLEEP_TIME = 2000; // 2 seconds
+
+    @Argument(index = 0, name = "sleep-time",
```

Comment:
Thinking about those that will have to type this command frequently -- Is there a way to define short-form arguments, if yes then `-t` would be nice. Also, let's make the unit explicit, preferably seconds (as a float, so we can define interval shorter than 1 second). Also, just `time` is fine. The fact that we sleep is an implementation detail.
```suggestion
    @Argument(index = 0, name = "time",
```

Reply:
Changes all positional arguments to options.